<launch>
    <arg name="bounding_box_depth_mapping" default="True"/>
    <arg name="rgb" default="True"/>
    <arg name="depth" default="True"/>
    <arg name="sync_nn" default="True"/>

    <node name="depthai_spatial_detection" pkg="cv" type="depthai_spatial_detection.py" output="screen">
        <param name="bounding_box_depth_mapping" value="$(arg bounding_box_depth_mapping)" />
        <param name="rgb" value="$(arg rgb)" />
        <param name="depth" value="$(arg depth)" />
        <param name="sync_nn" value="$(arg sync_nn)" />
    </node>
</launch>
